server/tests/test_background_tasks.py

```diff
@@ -278,7 +278,7 @@ def db_error():
             "startedAt": assert_is_date,
             "completedAt": assert_is_date,
             "error": asserts_startswith(
-                '(psycopg2.errors.NotNullViolation) null value in column "audit_name" violates not-null constraint'
+                '(psycopg2.errors.NotNullViolation) null value in column "audit_name"'
```

Comment:
The error message I was getting in my run was more explicit, talking about the table on which this error occurred. My interpretation is that this is a diff version of Postgres situation, and might as well make the test resilient to different versions of Postgres.